docs/en/02_Developer_Guides/10_Email/index.md

```diff
@@ -7,10 +7,16 @@ covers how to create an `Email` instance, customise it with a HTML template, the
 
 ## Configuration
 
-Out of the box, SilverStripe will use the built-in PHP `mail()` command. If you are not running an SMTP server, you 
-will need to either configure PHP's SMTP settings (see [PHP documentation](http://php.net/mail) to include your mail 
-server configuration or use one of the third party SMTP services like [SparkPost](https://github.com/lekoala/silverstripe-sparkpost) 
-and [Postmark](https://github.com/fullscreeninteractive/silverstripe-postmarkmailer).
+SilverStripe provides an API over the top of the [SwiftMailer](http://swiftmailer.org/) PHP library which comes with an
+extensive list of "transports" for sending mail via different services. 
+
+Out of the box, SilverStripe will use the built-in PHP `mail()` command via the `Swift_MailTransport` class. If you'd
+like to use a more robust transport to send mail you can swap out the transport used by the `Mailer` via config:
+
+```yml
+SilverStripe\Control\Email\Mailer:
+  swift_transport: Swift_SendmailTransport
```

Comment:
Would it be better to recommend that developers override the `Swift_Transport` service instead? Otherwise the resulting service definition list will include a service called "Swift_Transport"' that's not actually the swift transport being used...

```yml
Swift_Transport:
  class: Swift_SendmailTransport
```